src/test/java/impl/service/ReportGeneratorImplTest.java

```diff
@@ -0,0 +1,49 @@
+package impl.service;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import db.Storage;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReportGeneratorImplTest {
+    private ReportGeneratorImpl reportGenerator;
+
+    @BeforeEach
+    void setUp() {
+        Map<String, Integer> inventoryCopy = Storage.getAll();
+        for (String fruit : inventoryCopy.keySet()) {
+            Storage.set(fruit, 0);
+        }
+        reportGenerator = new ReportGeneratorImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Map<String, Integer> inventoryCopy = Storage.getAll();
+        for (String fruit : inventoryCopy.keySet()) {
+            Storage.set(fruit, 0);
+        }
```

Comment:
map has method clear, let's use it
```suggestion
        Storage.inventory.clear();
```